packages/tabs/src/Tab.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export interface TabProps {
+  /**
+   * Value supplied to the data-tab-id attribute. Used to determine what Tab is active.
+   */
+  value: string;
+
+  /**
+   * Content that will appear as the title in the Tab list.
+   */
+  tabTitle: string;
```

Comment:
Can this just be `title`? Reads redundant when written out: `<Tab tabTitle="Blah" />`.

Reply:
👍